src/main/Race.java

```diff
@@ -0,0 +1,64 @@
+package main;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "races")
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@NoArgsConstructor
+public class Race {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(unique = true, nullable = false)
+    Integer id;
+
+    @Column(nullable = false)
+    int round;
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    Team firstTeam;
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    Team secondTeam;
+
+    @Column(nullable = false)
+    String time;
+
+    public Race(int round, Team firstTeam, Team secondTeam) {
```

Comment:
You specified that `time` should be not null.
But here we see constructor that generates broken entity.